snuba/query/logical.py

```diff
@@ -428,6 +428,14 @@ def get_all_ast_referenced_columns(self) -> Set[Column]:
             ret |= {c for c in expression if isinstance(c, Column)}
         return ret
 
+    def get_columns_referenced_in_conditions_ast(self) -> Set[Column]:
```

Comment:
This is a copy paste of the previous method. Mind reusing this code by creating a private method that does this work and takes the expressions to look into as argument and call such method from get_all_ast_referenced_column and from this one ?